src/core/lib/security/credentials/tls/grpc_tls_credentials_options.h

```diff
@@ -123,7 +123,6 @@ struct grpc_tls_credentials_options
   // provider and the distributor, so we will get the distributor directly.
```

Comment:
Could you please also change the comments, if it's not the case any more? Thanks!

Reply:
Done

Reply:
Great, thanks!